Under The Weather/SceneDelegate.swift

```diff
@@ -20,7 +21,8 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         let dependencyContainer = DependencyContainer()
         let navigator = AppNavigator(window: window!)
         let coordinator = AppCoordinator(navigator: navigator, factory: dependencyContainer)
-        coordinator.start(animated: true, onDismissed: nil)
+        let navigationController = BasicNavigationController()
```

Comment:
navigationController is instantiated here but we don't use it?

Reply:
Either the linter or compiler should have caught that. Good catch, I'll remove it in the next commit 👍🏽